R/paginate.R

```diff
@@ -106,15 +120,15 @@ paginate_req_perform <- function(req,
   )
 
   for (page in seq2(2, n_pages)) {
-    req <- paginate_next_request(resp, req)
+    req <- paginate_next_request(resp, req, body)
     if (is.null(req)) {
       page <- page - 1L
       break
     }
 
     resp <- req_perform(req)
+    body <- paginate_parse_body(resp, req)
 
-    body_parsed <- resp_body_json(resp)
     out[[page]] <- resp
```

Comment:
I wonder if we should use the parsed object here too?

Reply:
Sounds very reasonable to me. Also after renaming to `parse_resp` this sounds a) general enough and b) clear enough to kind of expect this.